lib/template_filter.rb

```diff
@@ -7,17 +7,15 @@ def apply_scoping(scope, customizable = false, all = false)
       # Include the default template in the list of funder templates
       orgs << Template.default.org unless current_user.org == Template.default.org
 
-      templates = Template.unarchived
-        .where(family_id: Template.families(orgs.map(&:id)).pluck(:family_id))
-        .where(published: true)
-        .where(visibility: Template.visibilities[:publicly_visible])
-        .includes(:org)
+      # Gather all of the unarchived published publicly visible templates
+      # for the specified orgs
+      templates = Template.includes(:org).families(orgs.collect(&:id)).
```

Comment:
If I'm not missing anything templates within this block is only used for collecting the family_ids since the else i.e. !customizable reinitialises it again, what about:
```
templates = Templates.families(orgs.map(&:id).pluck(:family_id)
```
if not, where is used the org pre-fetched?